server/db/models/orders.js

```diff
@@ -0,0 +1,24 @@
+// define Orders model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
+
+const { UUID, UUIDV4, STRING, DECIMAL } = Sequelize;
+
+const Orders = db.define('orders', {
+    orderId: {
+        primaryKey: true,
+        type: UUID,
+        defaultValue: UUIDV4,
+    },
+    orderCost: {
+        //how to make order cost as the sum
+        //of product cost?
+        type: DECIMAL(10, 2),
+        allowNull: false,
+    },
+    shippingAddress: {
```

Comment:
is this allowed to be null?

Reply:
I think so. The shippingAddress can be found in the Users model once I connect the 2 models. But if we allow user to set more than 1 shipping address, this column will have some data.

Reply:
I think once an order is submitted it would HAVE to have a shipping address because where would we send the order to? I think a User model could only provide a shipping address if they have one on file like if they are signed in or have made a previous order.

Reply:
When a guest account user types in a shipping address, the address can be saved in User model, right?

Reply:
when a brand new guest with no shipping address saved in our database comes to our site and submits an order, the post request will be through the /api/orders route and create a new order.  I'm thinking it would make a put to request to through the /api/user route as well to update the shipping address so we can use later.

We can validate the shipping address as required on the client side but regardless a shipping address for an order that needs to be sent shouldn't be null under any circumstances.

Reply:
Ok, I'm going to add the shipping address to the Order model and make it not null.

Reply:
@wulalasuperdope . I'm currently working off of your branch, so can you hold off? Otherwise we'll hit merge conflicts.

Reply:
Sure